Maddori.Apple/Maddori.Apple/Screen/Setup/JoinTeamViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  JoinTeamViewController.swift
+//  Maddori.Apple
+//
+//  Created by LeeSungHo on 2022/10/20.
+//
+
+import UIKit
+
+import SnapKit
+
+final class JoinTeamViewController: BaseTextFieldViewController {
+    
+    var name = "진저"
+    
+    override var titleText: String {
+        get {
+            return name + TextLiteral.joinTeamViewControllerTitleLabel
+        }
+        
+        set {
+            super.titleText = newValue
+        }
+    }
+    
+    override var placeholerText: String {
+        get {
+            return TextLiteral.joinTeamViewControllerNicknameTextFieldPlaceHolder
+        }
+        
+        set {
+            super.placeholerText = newValue
+        }
+    }
+    
+    // MARK: - property
+    let createView: LabelButtonView = {
```

Comment:
p4;

```suggestion
    // MARK: - property
    
    let createView: LabelButtonView = {
```
요기도요기도... 하하하ㅏ하하ㅏ하ㅏㅏ

Reply:
[e2917b0](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/32/commits/e2917b0c4f46f2a200cd26b8642c1eed2bbea553)